### PR TITLE
Add munin plugin for minion jobs

### PR DESCRIPTION
**Pull request by perlpunk**

### CODE REVIEW

File: tools/munin/minion [hunk 1]
```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env perl
```
Comment (okurz) [inline tools/munin/minion:1]: IMHO tools are for developing openQA. How about "contrib/"?

File: tools/munin/minion [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+#openqa_minion_jobs,url=https://openqa.opensuse.org active=0i,delayed=0i,failed=352i,inactive=0i
+#openqa_minion_jobs_hook_rc_failed,url=https://openqa.opensuse.org rc_failed_per_10min=0i 16861374000000000
+#openqa_minion_workers,url=https://openqa.opensuse.org active=0i,inactive=1i,registered=1i
+my @data = qx{curl -s $url};
+
+my %values;
+for my $line (@data) {
+    my ($metric, $url, $values) = $line =~ m/^(\w+),url=(\S+) (\S+)/;
+    next unless $metric eq "openqa_$category";
+    %values = map { s/i$//r } map { split m/=/, $_ } split m/,/, $values;
```
Comment (kalikiana) [inline tools/munin/minion:59]: Can we have a simple unit test for this?

Comment (perlpunk) [reply tools/munin/minion]: I added a test

File: tools/munin/minion (hunk 1 above)
Comment (perlpunk) [reply tools/munin/minion]: I moved the script to `contrib/munin/`

File: contrib/munin/minion [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    # openqa_minion_jobs,url=https://openqa.opensuse.org active=0i,delayed=0i,failed=352i,inactive=0i
+    # openqa_minion_jobs_hook_rc_failed,url=https://openqa.opensuse.org rc_failed_per_10min=0i 16861374000000000
+    # openqa_minion_workers,url=https://openqa.opensuse.org active=0i,inactive=1i,registered=1i
+    my @data = qx{curl -s $url};
+
+    my %values;
+    for my $line (@data) {
+        my ($metric, $url, $values) = $line =~ m/^(\w+),url=(\S+) (\S+)/;
+        next unless $metric eq "openqa_$category";
+        %values = map { s/i$//r } map { split m/=/, $_ } split m/,/, $values;
```
[the remark this answers is not in the document]
Comment (perlpunk) [reply contrib/munin/minion:65]: I added a `last`

File: contrib/munin/minion [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+    for my $state (@$states) {
+        my $value = $values{ $state } // 0;
+        $total += $value;
+        print <<"EOM";
+$state.value $value
+EOM
+    }
+    print <<"EOM";
+total.value $total
+EOM
```
[the remark this answers is not in the document]
Comment (perlpunk) [reply contrib/munin/minion]: I changed it.
I like the heredocs though so it is easy to spot where we have munin output.

File: t/46-contrib.t [hunk 5]
```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env perl
```
[the remark this answers is not in the document]
Comment (perlpunk) [reply t/46-contrib.t]: renamed

File: t/46-contrib.t [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+# Copyright 2020 SUSE LLC
```
[the remark this answers is not in the document]
Comment (perlpunk) [reply t/46-contrib.t]: done

**Review by Martchus — APPROVED** [review]

**Review by asdil12 — APPROVED** [review]

**Review by okurz — APPROVED** [review]
Missing OBS checks (as of now)

**Pull request merged**